app/views/layouts/admin.html.erb

```diff
@@ -81,7 +81,7 @@
     </div>
     <main id="main-content" class="crayons-layout__content min-w-0 p-2">
       <% flash.each do |type, message| %>
-        <div class="alert alert-<%= type == "notice" || type == "success" ? "success" : "danger" %>">
+        <div class="alert alert-<%= type == "notice" || type == "success" ? "success" : "danger" %>" aria-live="<%= type == "danger" ? "assertive" : "polite" %>">
```

Comment:
I opted for `role="assertive"` when flash messages are of type `"danger"`. Otherwise the flash message receives the `role="polite"` attribute.

Reply:
I'm not sure this works unfortunately (I was just looking at it too 😄 ) - the messages still aren't announced... I think it's because `aria-live` is used to announce _dynamic_ changes to its content. The way these flash messages work, they're in the UI from the moment the page loads, so I think screen readers are just ignoring them..

from the MDN docs:

> works as long as you add the attribute before the changes occur — either in the original markup, or dynamically using JavaScript.

Reply:
Good point. 😎

Reply:
I'm going to revert to data-testid for now and once we have a better accessible solution in place, the tests can be updated to reflect that.

Reply:
It's using the `data-testid` attribute again and like I said, we'll reassess once a more accessible solution is in place.